[CUBRIDMAN-294] [KO][EN] SP 실행 시 성능 최적화와 TRACE 에 대한 설명 추가 (#627)
CUBRID/cubrid-manual: en/sql/tuning.rst

*   fetch_time : total fetch time when this query is performed(ms)
*   ioread: total I/O read count about this query. disk access count when the data is read

**FUNC**: This is displayed when stored procedures are called in the query.

*   time: total execution time of the stored procedures in this query(ms)

> *   fetch: the number of times pages are fetched during the stored procedures call in this query

Reviewer: 한글 매뉴얼에 언급한 내용을 영문에도 반영부탁합니다.

> *   time: total execution time of the stored procedures (ms)

Reviewer 2: ```suggestion
*   time: total execution time of the stored procedure (ms)
```
's'는 있거나 없어도 되지만 위의 'the stored procedure'랑 일관성있게 's'를 제거하는게 좋을 듯 합니다.

> *   fetch: the number of times pages are fetched during the stored procedures call

Reviewer 2: ```suggestion
*   fetch: the number of times pages are fetched during the stored procedure call
```

> *   ioread: total I/O read count during the stored procedures call. disk access count when the data is read

Reviewer 2: ```suggestion
*   ioread: total I/O read count during the stored procedure call; indicates the number of disk accesses when reading data
```
